src/transactions/TransactionUtils.h

```diff
@@ -15,6 +15,12 @@ class AbstractLedgerTxn;
 class LedgerTxnEntry;
 class LedgerTxnHeader;
 class TrustLineWrapper;
+struct LedgerKey;
+
+LedgerKey getLedgerKey(AccountID const& accountID);
```

Comment:
The fact that these 4 overloads make keys for accounts, trustlines, offers, and data seems a little non-obvious from their signatures & names. Maybe name them differently (eg. `getLedgerKeyForAccount`, `getLedgerKeyForTrustLine`, ...)?

Reply:
Sure. I think @vogel didn't like these names very much also. Maybe we can remove `get` and add entity names, like `ledgerKeyForAccount`, `ledgerKeyForTrustline`, etc.

Reply:
I would vote for either just ledgerKey(..) for everything or accountKey, trustlineKey, offerKey...

Reply:
ok, I went with Rafal's suggestion, hope that makes everyone happy :)